docs/tutorials/operations.md

```diff
@@ -0,0 +1,166 @@
+part of running as an [API Gateway with an East-West
+setup](../../kubernetes/ingress-controller/#run-as-api-gateway-with-east-west-setup).
+You have to add `-enable-kubernetes-east-west` and optional choose the
+domain of your choice
+`-kubernetes-east-west-domain=.ingress.cluster.local`. Be warned: There is a
+[known bug](https://github.com/zalando/skipper/issues/1024), if you
+combine it with custom routes.
+
+As part of API Gateway features, skipper supports [API
+monitoring](https://opensource.zalando.com/skipper/reference/filters/#apiusagemonitoring)
+and common [authentication and
+authorization](https://opensource.zalando.com/skipper/tutorials/auth/)
+protocols in Microservices architectures.
+
+Skipper has support for different [OpenTracing API](http://opentracing.io/) vendors, including
+[jaeger](https://www.jaegertracing.io/),
+[lightstep](https://lightstep.com/) and
+[instana](https://www.instana.com/supported-technologies/opentracing/).
+For example to configure the lightstep opentracing plugin, with a
+searchable `component` and `cluster` `tag` you can use:
+`- "-opentracing=lightstep component-name=skipper-ingress token=$(LIGHTSTEP_TOKEN) collector=tracing-collector.endpoint:8444 cmd-line=skipper-ingress max-buffered-spans=4096 tag=cluster=mycluster"`.
+The `LIGHTSTEP_TOKEN` is passed as environment variable to
+the process.
+
+Skipper can also add [global default filters](../../operation/operation/#global-default-filters),
+which will be automatically added to all routes. For example you can
+use `-default-filters-prepend="enableAccessLog(4,5)"` to enable only
+access logs in case of HTTP codes 4xx or 5xx. In the specific case of
+`*AccessLog` filters and `-default-filters-prepend`, the default
+choice can be overridden by users via `zalando.org/skipper-filter`
+ingress annotation.
+
+### Production example
+
+A full production deployment example you find at [Zalando's configuration
+repository](https://github.com/zalando-incubator/kubernetes-on-aws/tree/beta/cluster/manifests/skipper).
+
+### Recommendations
+
+We recommend to run a loadbalancer in front of Skipper to terminate
+TLS, such that cluster users can not access your keys and
+certificates. While skipper supports SNI, hardware and cloud
+loadbalancers often have hardware support to terminate TLS. It's
+cheaper for you to offload TLS to these devices and trust your compute
+vendor.
+
+We recommend to start [simple](#first-steps) and grow the feature set from there.
+Check features, that are used in `>60` production clusters in [Zalando's configuration
+repository](https://github.com/zalando-incubator/kubernetes-on-aws/tree/beta/cluster/manifests/skipper).
+
+#### Dashboards
+
+As an operator, build a Skipper dashboard and learn how Skipper and
+the Go runtime behaves with your workload. We successfully ran several
+load tests from 0 to 25k requests per seconds.  The load test was
+ramp up in less than a minute with initially 3 Skipper Pods, with an
```

Comment:
"The load test was ramp up..." -> "The load test was ramping up..."

Reply:
fixed